Automattic/wp-calypso
# Update WoA cancellation warning copy

#### Proposed Changes

* Updated WoA cancellation warning copy to let the customer know we will set their site to private.
* A couple grammatical edits.

Before | After
--|--
![before-update](https://user-images.githubusercontent.com/140841/173878689-c018cf7a-4b4e-47de-af9a-79893c5c6125.png)  | ![after-editorial](https://user-images.githubusercontent.com/140841/174651118-e62dfc6d-53aa-466c-ba02-36c7c6eeabe4.png)

#### Testing Instructions

Sandbox this PR and cancel a test site that is on the Atomic platform. Before completing the cancellation process, you will be presented with the warning above. Confirm it is the updated copy, grammatically correct, and delivers the message.

Related to #57278
